scripts/py_matter_idl/matter_idl/zapxml/handlers/handlers.py

```diff
@@ -483,12 +461,9 @@ def GetNextProcessor(self, name: str, attrs):
             return AttributeHandler(self.context, self._cluster, attrs)
         elif name.lower() == 'event':
             return EventHandler(self.context, self._cluster, attrs)
-        elif name.lower() == 'globalattribute':
-            # We ignore 'side' and 'value' since they do not seem useful
-            return ClusterGlobalAttributeHandler(self.context, self._cluster, ParseInt(attrs['code']))
         elif name.lower() == 'command':
             return CommandHandler(self.context, self._cluster, attrs)
-        elif name.lower() in ['define', 'description', 'domain', 'tag', 'client', 'server']:
+        elif name.lower() in ['define', 'description', 'domain', 'tag', 'client', 'server', 'globalattribute']:
```

Comment:
consider adding a TODO or not removing the old class as it seems globalattribute in clusters provide:
   - default value
   - feature bits for featureflags (unsure how we would model those ... but potentially possible)

https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml#L45

The default values seem somewhat useful if we ever ingest xmls instead of .matter files.

Reply:
I have updated the code to keep the class but ensure that the `GlobalHandler` pre-processor function is runned first.